src/pages/Lending/Pool/components/LoanSummary.tsx

```diff
@@ -97,16 +129,36 @@ export const LoanSummary: React.FC<LoanSummaryProps> = ({
   const useLendingQuoteQueryArgs = useMemo(
     () => ({
       collateralAssetId,
-      borrowAssetId,
-      depositAmountCryptoPrecision,
+      borrowAssetId: borrowAssetId ?? '',
+      depositAmountCryptoPrecision: depositAmountCryptoPrecision ?? '0',
     }),
     [collateralAssetId, borrowAssetId, depositAmountCryptoPrecision],
   )
   const {
     data: lendingQuoteData,
     isLoading: isLendingQuoteLoading,
     isError: isLendingQuoteError,
-  } = useLendingQuoteQuery(useLendingQuoteQueryArgs)
+  } = useLendingQuoteOpenQuery(useLendingQuoteQueryArgs)
+
+  const useLendingQuoteCloseQueryArgs = useMemo(
+    () => ({
+      collateralAssetId,
+      repaymentAssetId: repaymentAsset?.assetId ?? '',
+      repaymentPercent: Number(repaymentPercent),
```

Comment:
Making tsc happy, since according to the discriminated union, this can be undefined - it's already a number for the repay flow so the casting is effectively a no-op

Reply:
ty for the comment, I was just about to ask this! Silly TS.